formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt

```diff
@@ -119,9 +119,9 @@ internal class FragmentFlowRenderView(
         activity.supportFragmentManager.registerFragmentLifecycleCallbacks(callback, false)
     }
 
-    override val render: Renderer<FragmentFlowState> = Renderer {
```

Comment:
Removing `Renderer` here as equality check on `FragmentFlowState` is not necessary

Reply:
Will this have an effect downstream? Should be transparent, right?

Reply:
Individual renderers do an equality check by default, so this extra check does nothing